css-nesting-1/Overview.bs

```diff
@@ -466,11 +466,12 @@ Nesting Conditional Rules {#conditionals}
 -----------------------------------------------
 
 	A style rule can have any number of <a>conditional rules</a> inside of it, of any type,
-	intermixed with any number of declarations, in any order.
+	with rules required to come after declarations (see <a href="#mixing">Mixing Nesting Rules and Declarations</a>).
 
-	The presence of a nested conditional engages the same logic as if ''@nest'' was present.
-	The nested conditional rules must contain a <a>nesting selector</a> and follow the same
-	rules as outlined in <a href="#direct">direct nesting</a>.
+	If there are only rules in the nested conditional, the nesting context is adopted.
```

Comment:
Do you mean:
```suggestion
	If there are only declarations in the nested conditional, the nesting context is adopted.
```
?

But even so, why restrict it that way? if we have mixed rules and declarations, we can still adopt the nesting context. E.g. this:

```css
.fancy-link {
	color: purple;

	@supports (fancy-shadow: purple) {
		color: transparent;
		fancy-shadow: purple;

		&:hover {
			color: mix(purple, white);
		}
	}
}
```

can be sugar for:

```css
.fancy-link {
	color: purple;

	@supports (fancy-shadow: purple) {
		& {
			  color: transparent;
			  fancy-shadow: purple;
	  
			  &:hover {
				  color: mix(purple, white);
			  }
		}
	}
}
```

Even though the conditional rule contains both rules and declarations.